etc/kayobe/ansible/growroot.yml

```diff
@@ -27,57 +27,70 @@
     growroot_vg: "rootvg"
     # Don't assume facts are present.
     os_family: "{{ ansible_facts.os_family | default('Debian' if os_distribution == 'ubuntu' else 'RedHat') }}"
+    # LVM override
+    skip_lvm_check: false
```

Comment:
Might be worth prefixing this with something like `growroot_` to make it a bit more specific

Reply:
Done :)